resources/definitions/GEEETECH_A10.def.json

```diff
@@ -0,0 +1,57 @@
+{
+    "version": 2,
+    "name": "GEEETECH A10",
+    "inherits": "fdmprinter",
+    "metadata": {
+        "visible": true,
+        "author": "Amit L",
+        "manufacturer": "GEEETECH TECH.",
+        "file_formats": "text/x-gcode",
+        "has_materials": true,
+        "machine_extruder_trains":
+        {
+            "0": "GEEETECH_A10_1"
+        }
+
+    },
+
+    "overrides": {
+        "machine_name": { "default_value": "GEEETECH A10" },
+        "machine_width": {
+            "default_value": 220
+        },
+        "machine_height": {
+            "default_value": 220
+        },
+        "machine_depth": {
+            "default_value": 220

```

Comment:
In the [specifications](https://www.geeetech.com/geeetech-a10-open-source-big-building-volume-3d-printer-p-1033.html) they say that it's 220x220x2*6*0mm build volume. Should this be changed or was that intentional?